Standards/scs-0103-v1-standard-flavors.md

```diff
@@ -14,6 +14,19 @@ description: |
 
 ## Introduction
 
+This is v1.1 of the standard, which lifts the following restriction regarding the property `scs:name-vN`:
+this property may now be used on any flavor, rather than standard flavors only. In addition, the "vN" is
+now interpreted as "name variant N" instead of "version N of the naming standard". Note that this change
+indeed preserves compliance, i.e., compliance with v1.0 implies compliance with v1.1.
+
+## Terminology
+
+extra_specs
+  Additional properties on an OpenStack flavor, see
+  https://docs.openstack.org/nova/2024.1/user/flavors.html#extra-specs
+  and
+  https://docs.openstack.org/nova/2024.1/configuration/extra-specs.html
```

Comment:
```suggestion
  <https://docs.openstack.org/nova/2024.1/user/flavors.html#extra-specs>
  and
  <https://docs.openstack.org/nova/2024.1/configuration/extra-specs.html>
```

should suffice to please the markdown linter gods

Reply:
I think this linter is outdated. The newer Docusaurus prefers bare links.

Reply:
@maxwolfs can you please comment on this?

Reply:
I have removed the bare links by adding descriptions.